src/crypto/crypto_sig.cc

```diff
@@ -274,23 +271,12 @@ void CheckThrow(Environment* env, SignBase::Error error) {
 }
 
 bool IsOneShot(const EVPKeyPointer& key) {
-  switch (EVP_PKEY_id(key.get())) {
-    case EVP_PKEY_ED25519:
-    case EVP_PKEY_ED448:
-      return true;
-    default:
-      return false;
-  }
+  return key.id() == EVP_PKEY_ED25519 || key.id() == EVP_PKEY_ED448;
```

Comment:
all of these functions can be constexpr

Reply:
Let's separately go through and make these kinds of improvements in one of the follow up PRs